### ISSUE TITLE
ground truth label problem

### ISSUE THREAD

**Issue by elvindp** [WXinlong/SOLO#200]
in 'mmdet/models/anchor_heads/solov2_head.py', function 'solov2_target_single' generate ground truth label, but I find that,
when initial cate_label, the value is 0:
            cate_label = torch.zeros([num_grid, num_grid], dtype=torch.int64, device=device)
and then update cate_label, if center in that grid:
            cate_label[top:(down+1), left:(right+1)] = gt_label
but the class label is starting with 0th class, this will product a lot of 0th class label.
I quite confused. Could anyone help me?
